packages/automerge-repo-react-hooks/src/useRemoteAwareness.ts

```diff
@@ -31,30 +31,30 @@ export type Heartbeats = Record<string, number>
  * It also returns their heartbeat status.
  * It is intended to be used alongside useLocalAwareness.
  *
- * @param {string} props.handle A document handle to associate with
- * @param {string?} props.localUserId Automerge BroadcastChannel sometimes sends us our own messages; optionally filters them
- * @param {number?30000} props.offlineTimeout How long to wait (in ms) before marking a peer as offline
- * @param {function?} props.getTime Function to provide current epoch time (used for testing)
- * @returns [ peerStates: { [userId]: state, ... }, { [userId]: heartbeatEpochTime, ...} ]
  */
 export const useRemoteAwareness = <T>({
   handle,
   localUserId,
   offlineTimeout = 30000,
   getTime = () => new Date().getTime(),
-}: UseRemoteAwarenessProps<T>): [PeerStates, Heartbeats] => {
+}: UseRemoteAwarenessProps<T>): [PeerStates<T>, Heartbeats] => {
   // TODO: You should be able to use multiple instances of this hook on the same handle (write test)
   // TODO: This should support some kind of caching or memoization when switching between channelIDs
-  const [peerStates, setPeerStates, peerStatesRef] = useStateRef<PeerStates>({})
+  const [peerStates, setPeerStates, peerStatesRef] = useStateRef<PeerStates<T>>(
+    {}
+  )
   const [heartbeats, setHeartbeats, heartbeatsRef] = useStateRef<Heartbeats>({})
   useEffect(() => {
+    if (!handle) return
```

Comment:
separate commit? also add braces

Reply:
I personally prefer braces, but if it passes linting, this is okay by me. We can litigate that separately. No need for a separate commit.